PyInstaller/loader/pyimod03_importers.py

```diff
@@ -234,7 +234,7 @@ def __init__(self):
 
     def __call__(self, path):
         """
-        PEP-302 sys.path_hook processor.
+        PEP-302 sys.path_hook processor. Only needed for Python 2.
```

Comment:
Please explain briefly why only used for Python 2.

Reply:
Done.